src/ObjectUtils/SymbolsFile.cpp

```diff
@@ -52,4 +54,29 @@ ErrorMessageOr<std::unique_ptr<SymbolsFile>> CreateSymbolsFile(
   return ErrorMessage{error_message};
 }
 
+[[nodiscard]] std::vector<std::filesystem::path> GetAllowedSymbolFilenamesForModule(
```

Comment:
Nit: To me "Allowed" gives the wrong impression. It's not that we don't allow other filename. We are just not looking for them, right? What about `GetTypicalSymbolFilenamesForModule` or `GetStandardSymbolFilenamesForModule`?

Reply:
done -> `GetStandardSymbolFilenamesForModule`